### PR TITLE
Add Vue.js form handling example

### PR DESCRIPTION
**Pull request by luixlacrux**
Addresses #48 

- [x] Add Vue.js form handling

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1852,7 +1852,43 @@ export default class ReactForm extends React.Component{
...
+export default {
+  data() {
+    return {
+      email: '',
+      password: ''
+    }
+  },
+  methods: {
+    onSubmit() {
+      Auth.sign(this.email, this.password);
```
Comment (feimosi) [inline README.md]: `Auth.signIn`

File: README.md [hunk 2]
```diff
@@ -1852,7 +1852,43 @@ export default class ReactForm extends React.Component{
 
 ### Vue.js
 
-> TODO
+You can use the `v-model` directive to create **two-way data bindings** on form `input` and `textarea` elements. It automatically picks the correct way to update the element based on the input type. Although a bit magical, `v-model` is essentially syntax sugar for updating data on user input events, plus special care for some edge cases.
```
Comment (feimosi) [inline README.md]: Could you add this paragraph to the `Data binding` section?

**Review by feimosi — CHANGES_REQUESTED** [review]
Nice, thanks!

File: README.md (hunk 2 above)
Comment (luixlacrux) [reply README.md]: I Remove it from here and I add it to the `Data binding` section?

File: README.md (hunk 1 above)
Comment (luixlacrux) [reply README.md]: Oh, thanks for the suggest

File: README.md (hunk 2 above)
Comment (feimosi) [reply README.md]: Just the text about `v-modal` should be moved to `Data binding` -> `Vue.js`. The rest stays here.

**Comment by feimosi** [comment]
Looks good :+1:

**Pull request merged**